### DISCUSSION TITLE
Currency Conversion Calculator.xaml

### DISCUSSION THREAD

**Discussion by maryamabasova** [LianqiangGuan/Tafe230211#4] [General]
Hello Team, I wanted to add Currency Conversion Calculator.xaml in my Visual Studio 2022 but it shows an error. What do you think I should do? Your help is much appreciated.

**Comment by LianqiangGuan** [comment]
Hello Maryam. You should create a new Blank Page in C# and then rename it to Currency_Conversion_Calculator.xaml, otherwise it reads it as a blank text file

**Reply by LianqiangGuan** [reply]
and the new version of Visual Studio have different window when you add new item

**Comment by billybena** [comment]
Steps to creating a new page:
1. open universal calculator solution
2. right click on Calculator (universal window) in solution explorer
3. select add new item and select UWP blank page
4. Rename page with you feature calculator name and create
